govcd/group.go

```diff
@@ -183,3 +183,21 @@ func validateDeleteGroup(group *types.Group) error {
 
 	return nil
 }
+
+// copyWithoutUserList returns a copy of the given group, with the UserList attribute set to nil.
+// This can and should be used to interact with VCD after a group read from the LDAP,
+// as having this list populated will return an error 400 as VCD doesn't expect this list to be updatable.
+func copyWithoutUserList(group *types.Group) *types.Group {
```

Comment:
Could you put one test step for this  function.  (could live in test_GroupUserListIsPopulated).


Something like this:
```go
originalGroup.Userlist = nil
check.Assert(copiedGroupWithoutUserList, DeepEquals, originalGroup)
```

This should allow us to remind improving this function if we ever add additional fields to `types.Group`.

Reply:
Done, ran the test with the new addition and it passes, please check now.